cms/templates/activation_invalid.html

```diff
@@ -17,7 +17,7 @@ <h1 class="page-header">${_("{studio_name} Account Activation").format(studio_na
       <div class="msg">
         <h1 class="title">${_("Your account activation is invalid")}</h1>
         <div class="copy">
-          <p>${_("We're sorry. Something went wrong with your activation. Check to make sure the URL you went to was correct — e-mail programs will sometimes split it into two lines.")}</p>
```

Comment:
Yes, that makes sense. Good catch. Let's actually change this to,
`correct, as e-mail`